server/lib/resolvers/builders/postgres.js

```diff
@@ -1,10 +1,12 @@
 const JSONParse = require('json-parse-safe')
 const { auditLog } = require('../../../lib/util/logger')
+const { updateResolverMetrics } = require('../../../metrics')
 
-function buildPostgresResolver (dataSource, compiledRequestMapping, compiledResponseMapping) {
+function buildPostgresResolver (dataSource, compiledRequestMapping, compiledResponseMapping, resolverMapping) {
```

Comment:
@psturc I know I suggested this, but I didn't update you with the latest approach to get the type, field name etc.

If you have a look at https://github.com/aerogear/data-sync-server/blob/master/server/lib/resolvers/builders/postgres.js#L33 you will see an `info` var is passed to auditLog function.

And, here,
https://github.com/aerogear/data-sync-server/blob/master/server/lib/util/logger.js#L11
in the auditLog function, we're able to get all the information related to Graphql resolver.

I found it better to get information from `info` object like that. Could you adapt the code to get type and field name from there?

There is additional information as well, if you like to add more labels to Prom metric:
* info.operation.operation : query/mutation
* info.fieldname: field name (a. createMeme or b. memes)
* info.parentType.name: parent type (a. Mutation or b. Profile)
* info.path: the whole path for the resolver (a. Mutation.createMeme or b. allProfiles.0.memes)

In my examples above, (a) is a top level mutation.
(b) is a inner resolver (no top level) that is executed when `memes` field of a `Profile` is to be resolved.

let me know if you need any help with this.

Reply:
Oh my, I have the audit logs in front of my eyes when working on this and I didn't realise! Thanks for the update! 
Should be no problem to rewrite this. I have the third part completed, so I will send suggested changes in the last commit.